app/_hub/kong-inc/ai-semantic-cache/how-to/llm-provider-integration-guides/_mistral.md

```diff
@@ -0,0 +1,73 @@
+---
+nav_title: Mistral
+title: Set up AI Semantic Cache with Mistral
+---
+
+## Prerequisites 
+
+* Mistral's API key
+* [Redis configured as a vector database](https://redis.io/docs/latest/develop/get-started/vector-database/)
+* [Redis configured as a cache](https://redis.io/docs/latest/operate/oss_and_stack/management/config/#configuring-redis-as-a-cache)
+* You need a service to contain the route for the LLM provider. Create a service **first**:
+  ```sh
+  curl -X POST http://localhost:8001/services \
+  --data "name=ai-semantic-cache" \
+  --data "url=http://localhost:32000"
+  ```
+  Remember that the upstream URL can point anywhere empty, as it won’t be used by the plugin.
+
+## Steps
+1. Create a route:
+```sh
+curl -X POST http://localhost:8001/services/ai-semantic-cache/routes \
+  --data "name=mistral-semantic-cache" \
+  --data "paths[]=~/mistral-semantic-cache$"
+```
+
+1. Set the AI Semantic Cache plugin. This uses Mistral's API Key explicitly, but you can use an environment variable instead if you want.
+```sh
```

Comment:
We should use the `plugin_examples` block for this step, but we can do that as a future improvement - shouldn't prevent merging.

Reply:
👍 I created a separate Jira ticket to make this update: https://konghq.atlassian.net/browse/DOCU-4060